src/mono/sample/iOS-NativeAOT/Program.csproj

```diff
@@ -60,7 +59,7 @@
     DependsOnTargets="IlcCompile;CopyFilesToPublishDirectory">
 
     <PropertyGroup>
-      <AppDir>$(MSBuildThisFileDirectory)$(PublishDir)\app</AppDir>
+      <AppDir>$(MSBuildThisFileDirectory)$(OutputPath)\Bundle</AppDir>
```

Comment:
I missed this in the previous PR with changes to iOS (mono) sample. Why are we changing the name of the AppDir?

Reply:
We match it with the AppleBuild.targets and the Mono sample. In the previous PR, there was a change in the bundle path from `bin/publish/app` to `bin/ios-arm64/Bundle`.